app/routes/events/components/EventDetail/index.tsx

```diff
@@ -205,16 +205,29 @@ export default class EventDetail extends Component<Props, State> {
     }
 
     const color = colorForEvent(event.eventType);
+
     const onRegisterClick = currentUserFollowing
       ? () => unfollow(currentUserFollowing.id, event.id)
       : () => follow(currentUser.id, event.id);
+
     const currentMoment = moment();
+
     const activationTimeMoment = moment(event.activationTime);
+
     const eventRegistrationTime = activationTimeMoment.subtract(
       penaltyHours(penalties),
       'hours'
     );
+
     const registrationCloseTimeMoment = registrationCloseTime(event);
+
+    // The UserGrid is expanded when there's less than 5 minutes till activation
+    const minUserGridRows = currentMoment.isAfter(
+      activationTimeMoment.subtract(5, 'minutes')
+    )
+      ? 2
```

Comment:
Could this, and the value of the maxRows property passed in line 434 be combined into a const?

Reply:
Sure thing